pkg/ccl/backupccl/backup_job.go

```diff
@@ -86,6 +86,12 @@ var BackupCheckpointInterval = settings.RegisterDurationSetting(
 	"the minimum time between writing progress checkpoints during a backup",
 	time.Minute)
 
+var backupUseFollowerReads = settings.RegisterBoolSetting(
```

Comment:
do we need this for merging to master? The only reason to disable it would be "hidden bug," right?

Reply:
Yes, we do. We like having a toggle for any change both in case there is an unforeseen bug like you mention, but also just so we can rule such a thing out as well: if we see something behave unexpectedly in the next several months, we might wonder _if_ it is a hidden bug, or even perhaps not a bug but just a new way it behaves, and being able to toggle it off quickly is a way to cheaply _rule out_ that possibility.

Reply:
👍